ext/mysqli/tests/mysqli_insert_packet_overflow.phpt

```diff
@@ -71,10 +71,15 @@ memory_limit=256M
         printf("[011] Failed to change max_allowed_packet");
     }
 
-    if (!mysqli_query($link, "CREATE TABLE test(col_blob LONGBLOB) ENGINE=" . $engine))
-        printf("[012] [%d] %s\n", mysqli_errno($link), mysqli_error($link));
+    $table_name = 'test__mysqli_insert_packet_overflow';
```

Comment:
Why the double underscore?

Reply:
I am having trouble articulating why I did that. Maybe I wanted 'test' to appear distinct from the name of the test case?